spec/builtins.test.sh

```diff
@@ -209,7 +209,7 @@ umask | grep '[0-9]\+'  # check for digits
 ## status: 0
 
 #### set umask in octal
-rm -f $TMP/umask-one $TMP/umask-two
+rm $TMP/umask-one $TMP/umask-two
```

Comment:
This looks like you didn'dt merge https://github.com/oilshell/oil/commit/71cb657d78307938609ae87a742707815c9ca899

?

Reply:
There was a merge conflict but I picked the wrong one, will fix.